app/services/hyrax/derivative_bucketed_storage.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
```

Comment:
I would have subclassed the override here but `Valkyrie::Storage::Disk::BucketStorage` is a inner class. That makes me think it is best to replace instead of extend.